lib/cartopy/tests/crs/test_nearside_perspective.py

```diff
@@ -23,28 +23,48 @@
 
 from numpy.testing import assert_almost_equal
 
-from cartopy.tests.crs.test_geostationary import (GeostationaryTestsMixin,
-                                                  check_proj4_params)
+from cartopy.tests.crs.test_geostationary import check_proj4_params
 
-from cartopy.crs import NearsidePerspective
+import cartopy.crs as ccrs
 
 
-class TestEquatorialDefault(GeostationaryTestsMixin, object):
-    # Check that it behaves just like Geostationary, in the absence of a
-    # central_latitude parameter.
-    test_class = NearsidePerspective
-    expected_proj_name = 'nsper'
+class TestNearsidePerspective(object):
+    def test_default(self):
+        geos = ccrs.NearsidePerspective()
+        other_args = {'a=6378137.0', 'h=35785831', 'lat_0=0.0', 'lon_0=0.0',
+                      'x_0=0', 'y_0=0'}
+
+        check_proj4_params('nsper', geos, other_args)
+
+        assert_almost_equal(geos.boundary.bounds,
+                            (-5476336.098, -5476336.098,
+                             5476336.098, 5476336.098),
+                            decimal=3)
+
+    def test_eastings(self):
```

Comment:
Rename to `test_offset` to match the others.

Reply:
Done.